### ISSUE TITLE
Accessibility Review for Staging Review: VRE CH31

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22653]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Additional feedback
A11y specialists still strongly recommend that the VR&E orientation widget should be placed on its own single page separate from the wizard itself in the future. It would make for a more simple, elegant, and accessible solution that could be linked to independently.

**Comment by joshkimux** [comment]
<img width="544" alt="Screen Shot 2021-04-07 at 3 56 15 PM" src="https://user-images.githubusercontent.com/14154792/113926111-c97ba680-97b9-11eb-9227-09d3ad4dcb9b.png">

Content question for @mnorthuis , would users know what time zone this is in? Is

**Comment by joshkimux** [comment]
@jason-gcio and @sporeboy 
<img width="1275" alt="Screen Shot 2021-04-07 at 4 07 34 PM" src="https://user-images.githubusercontent.com/14154792/113927450-815d8380-97bb-11eb-802b-4ad066659c33.png">

Would love to talk to y'all about this page. I've spent a good hour trying to break this apart into separate tickets, but have been struggling. Some things I'd like to talk about include:

- Consistency with headings in the form
- Splitting the page apart to reduce cognitive load
- Simplifying the design

**Comment by jason-gcio** [comment]
@joshkimux - totally, I'll see if I can catch up with you tomorrow.

**Comment by mnorthuis** [comment]
> <img alt="Screen Shot 2021-04-07 at 3 56 15 PM" width="544" src="https://user-images.githubusercontent.com/14154792/113926111-c97ba680-97b9-11eb-9227-09d3ad4dcb9b.png">
> 
> Content question for @mnorthuis , would users know what time zone this is in?

I think @peggygannon would be better to answer this based on content standards.  She will be back in the office Monday.

**Comment by cohnjesse** [comment]
Going to write a comment to attempt to direct the list of tickets on this epic to the correct place they need to be handled.

1. [22846](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/22846) - This is a ticket for platform to fix, anything inside the sign on widget is maintained by the platform team. This is a good general rule for future tickets, the sign on widget is a pre-made widget that we simply use but we didn't build and we don't maintain. It was built and is maintained by the platform team.

2. [22824](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/22824) - I am going to go out on a limb here and say this should be fixed at the platform level. The fix that was done by the profile team and Erik is a page level fix for a site wide issue. We don't want to fix each instance of a site wide issue separately, we want to fix it at the root level so it fixes it everywhere. Since this is an issue with a platform wide component, it should be fixed at the platform level.

3. [22826](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/22826) - This is the same as the previous ticket, it should be fixed at the platform level. We don't want to implement page level fixes for platform level components as this will generate more code than is needed and make long term fixes less effective where page level fixes have been implemented.

**Comment by micahchiang** [comment]
#22827 - left a comment on the ticket, should potentially be closed, pending confirmation from Josh.
